main.cpp

```diff
@@ -15,9 +15,55 @@ Create a branch named Part2
     You'll need to insert the Person struct from the video in the space below.
  */
 
+    struct Foot
+    {
+        void stepForward();
+        int stepSize();
+    };
+
+    void Foot::stepForward()
+    {
+    // I have no clue what to put in here
+    }
+
+    int Foot::stepSize()
+    {
+    // I have no clue what to put in here
+    return 4;
+    }
 
+    struct Person
+    {
+        int age;
+        int height;
+        float hairLength;
+        float GPA;
+        unsigned int SATScore;
+        int distanceTraveled;
 
+        void run(int howFast, bool startWithLeftFoot);
 
+    };
+
+    void Person::run(int howFast, bool startWithLeftFoot)
+    {
+        Foot leftFoot;
+        Foot rightFoot;
```

Comment:
These did not live in the function in the video.  They need to be class member variables.
- [ ] revise